styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
```

Comment:
I guess an ideal "Issue" should contain a specific topic from which actionable items can be determined. Actionable items turn into code commits.